salt/metalk8s/defaults.yaml

```diff
@@ -51,4 +53,36 @@ kubelet:
     authorization:
       mode: AlwaysAllow
 
+ca:
+  cert:
+    name: /etc/kubernetes/pki/ca.crt
+    signing_private_key: /etc/kubernetes/pki/ca.key
+    CN: kubenetes
+    keyusage: "critical digitalSignature, keyEncipherment, keyCertSign"
+    basicConstraints: "critical CA:true"
+    days_valid: 3650
+  signing_policies:
+    api:
+      - minions: '*'
+      - signing_private_key: /etc/kubernetes/pki/ca.key
```

Comment:
This seems duplicated from above.

Reply:
Yes it's but that allow to generate a key and a cert manually then just give the path here to use them.

Reply:
Convention over configuration. Just like `kubeadm` does it: we expect a file in a specific hard-coded location. If it's not there, we generate it, otherwise we use it as-is. No need to be able to change these paths etc.